docs/site/docusaurus.config.js

```diff
@@ -1,26 +1,35 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-const lightCodeTheme = require("prism-react-renderer/themes/github");
-const darkCodeTheme = require("prism-react-renderer/themes/nightOwl");
-const math = require("remark-math");
-const katex = require("rehype-katex");
+import { themes } from "prism-react-renderer";
```

Comment:
Don't we also need the darkCodeTheme? Or the `themes` now includes both and we can later interchange?